[2014-04-24 10:09:12] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "approve/mainvoice" as such route does not exist.' in /var/www/arp_dev/vendor/symfony/routing/Symfony/Component/Routing/Generator/UrlGenerator.php:134 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('approve/mainvoi...', Array, true) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/helpers.php(696): Illuminate\Routing\UrlGenerator->route('approve/mainvoi...', 1045, true) #2 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(81): route('approve/mainvoi...', 1045) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #8 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1045') #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-04-24 10:14:08] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Relations\HasOne::$status' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:106 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(106): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 106, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->approve('1045') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('approve', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'approve', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1045') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-04-24 10:14:49] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Relations\HasOne::$status' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:106 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(106): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 106, Array) #1 [internal function]: Controllers\Admin\MainvoiceController->approve('1045') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('approve', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'approve', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1045') #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #12 {main} [] []
[2014-04-24 10:15:52] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Controllers\Admin\Session' not found' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:108 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-04-24 10:37:44] log.ERROR: exception 'InvalidArgumentException' with message 'The HTTP status code "1043" is not valid.' in /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php:425 Stack trace: #0 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(142): Symfony\Component\HttpFoundation\Response->setStatusCode('1043') #1 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/RedirectResponse.php(44): Symfony\Component\HttpFoundation\Response->__construct('', '1043', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(188): Symfony\Component\HttpFoundation\RedirectResponse->__construct('http://127.0.0....', '1043', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(117): Illuminate\Routing\Redirector->createRedirect('http://127.0.0....', '1043', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(209): Illuminate\Routing\Redirector->to('show/mainvoice', '1043') #5 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Facade::__callStatic('to', Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Redirect::to('show/mainvoice', '1043') #7 [internal function]: Controllers\Admin\MainvoiceController->approve('1043') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('approve', Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'approve', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1043') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 10:43:19] log.ERROR: exception 'InvalidArgumentException' with message 'The HTTP status code "1043" is not valid.' in /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php:425 Stack trace: #0 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(142): Symfony\Component\HttpFoundation\Response->setStatusCode('1043') #1 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/RedirectResponse.php(44): Symfony\Component\HttpFoundation\Response->__construct('', '1043', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(188): Symfony\Component\HttpFoundation\RedirectResponse->__construct('http://127.0.0....', '1043', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(117): Illuminate\Routing\Redirector->createRedirect('http://127.0.0....', '1043', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(209): Illuminate\Routing\Redirector->to('show/mainvoice', '1043') #5 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Facade::__callStatic('to', Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Redirect::to('show/mainvoice', '1043') #7 [internal function]: Controllers\Admin\MainvoiceController->approve('1043') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('approve', Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'approve', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1043') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 10:43:34] log.ERROR: exception 'InvalidArgumentException' with message 'The HTTP status code "1043" is not valid.' in /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php:425 Stack trace: #0 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(142): Symfony\Component\HttpFoundation\Response->setStatusCode('1043') #1 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/RedirectResponse.php(44): Symfony\Component\HttpFoundation\Response->__construct('', '1043', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(188): Symfony\Component\HttpFoundation\RedirectResponse->__construct('http://127.0.0....', '1043', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Redirector.php(117): Illuminate\Routing\Redirector->createRedirect('http://127.0.0....', '1043', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(209): Illuminate\Routing\Redirector->to('show/mainvoice', '1043') #5 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Facade::__callStatic('to', Array) #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(116): Illuminate\Support\Facades\Redirect::to('show/mainvoice', '1043') #7 [internal function]: Controllers\Admin\MainvoiceController->approve('1043') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('approve', Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'approve', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1043') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 10:46:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function invoicestatus() on a non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:107 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-04-24 11:41:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:42:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:44:33] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:45:08] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:30] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:35] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:35] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:35] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:47:35] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:48:47] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:49:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:49:28] log.ERROR: exception 'ErrorException' with message 'Undefined variable: allcurrencies' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:130 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(130): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 130, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-04-24 11:52:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 11:57:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:01:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:01:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:02:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:02:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:06:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:07:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:08:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 12:28:39] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "groupa" as such route does not exist.' in /var/www/arp_dev/vendor/symfony/routing/Symfony/Component/Routing/Generator/UrlGenerator.php:134 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('groupa', Array, true) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/helpers.php(696): Illuminate\Routing\UrlGenerator->route('groupa', Array, true) #2 /var/www/arp_dev/app/storage/views/c0f24373ddee7f0bbc547030172b52c4(34): route('groupa') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #8 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getEdit', Object(Illuminate\View\View)) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getEdit', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('2') #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} [] []
[2014-04-24 12:30:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\HttpException' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:728 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Illuminate\Foundation\Application->abort(403) #1 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\Facade::__callStatic('abort', Array) #2 /var/www/arp_dev/app/filters.php(98): Illuminate\Support\Facades\App::abort(403) #3 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request)) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('admin-auth', Object(Illuminate\Http\Request), Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'admin-auth', Object(Illuminate\Http\Request)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'getIndex') #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-04-24 14:46:15] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Controllers\Admin\Sentry' not found' in /var/www/arp_dev/app/controllers/admin/DashboardController.php:29 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-04-24 14:46:16] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Controllers\Admin\Sentry' not found' in /var/www/arp_dev/app/controllers/admin/DashboardController.php:29 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-04-24 15:56:49] log.ERROR: exception 'ErrorException' with message 'Undefined variable: invoice' in /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373:486 Stack trace: #0 /var/www/arp_dev/app/storage/views/cc228d75f3c20261dd87044a69c2c373(486): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 486, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getIndex', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
